openassessment/assessment/test/test_staff.py

```diff
@@ -445,7 +445,36 @@ def test_cancel_staff_workflow(self):
         tim_sub, tim = self._create_student_and_submission("Tim", "Tim's answer")
         workflow_api.cancel_workflow(tim_sub['uuid'], "Test Cancel", "Bob", {})
         workflow = StaffWorkflow.objects.get(submission_uuid=tim_sub['uuid'])
-        self.assertIsNotNone(workflow.cancelled_at)
+        self.assertTrue(workflow.is_cancelled)
+
+    def test_grading_statistics(self):
+        bob_sub, bob = self._create_student_and_submission("bob", "bob's answer")
+        course_id = bob['course_id']
+        item_id = bob['item_id']
+        tim_sub, tim = self._create_student_and_submission("Tim", "Tim's answer")
+        sue_sub, sue = self._create_student_and_submission("Sue", "Sue's answer")
+        stats = staff_api.get_staff_grading_statistics(course_id, item_id)
+        self.assertEqual(stats, {'graded': 0, 'ungraded': 3, 'in-progress': 0})
```

Comment:
Can you toss some "cancelled" logic into this test also? Also coverage of the timeout functionality.